node/lib/task.ts

```diff
@@ -511,7 +511,11 @@ export function stats(path: string): FsStats {
  * @returns   boolean 
  */
 export function exist(path: string): boolean {
-    return path && fs.existsSync(path);
+    try {
+        return path && fs.statSync(path) != null;
+    } catch (err) {
+        return false;
```

Comment:
Please identify what specific issue/case you're addressing which led you to want to change this code